server/database/fakeData/index.ts

```diff
@@ -0,0 +1,48 @@
+import sequelize from '../config/connection';
+// eslint-disable-next-line import/extensions
+import * as data from './data.json';
+
+import {
+  Campaign,
+  Donor,
+  Family,
+  Donation,
+  Contact,
+  Category,
+  Capon,
+} from '../models';
+
+const {
+  campaigns, capons, categories, messages, donations, donors, families,
+} = data;
+
+const buildData = async () => {
+  await sequelize.sync({ force: true });
+
+  if (process.env.NODE_ENV === 'dev') {
```

Comment:
What if the env is not a development? You still need to build your fake data. The difference is when you are in a development you run your script to build the data but the script just reads this file which we define a function but we don't use it so we need to invoke it, whereas in the test env we invoke this function in our test files so we don't need to invoke it here again, that why you need to move this condition to the line where you invoke the function.